src/Nethermind/Nethermind.Consensus/AlwaysPoS.cs

```diff
@@ -23,9 +23,8 @@ private AlwaysPoS() { }
 
     public long? ConfiguredTerminalBlockNumber => 0;
 
-#pragma warning disable CS0067
-    public event EventHandler TerminalBlockReached;
-#pragma warning restore CS0067
+    public event EventHandler TerminalBlockReached { add { } remove { } }
+    public event EventHandler Transitioned { add { } remove { } }
```

Comment:
Is this required? Why not just `TerminalBlockReached`?

Reply:
Wanted to have transition finalized to minimize chance of reorg before enabling eth/69.